docs/README.md

```diff
@@ -1,20 +1,27 @@
 # Overview
 ![k0s logo](img/k0s-logo-full-color.svg)
 
-k0s is an all-inclusive Kubernetes distribution with all the required bells and whistles preconfigured to make building a Kubernetes clusters a matter of just copying an executable to every host and running it.
+k0s is an all-inclusive Kubernetes distribution, configured with all of the
```

Comment:
As a general comment for all document files, I think there shouldn't be manual line changes (enters) in the middle of the paragraphs (like done with this change). If the paragraph is written without enters it scales well for different display sizes. One can read this document for example with a small mobile device and the line width is adjusted automatically.

Reply:
Understood.